Neos.ContentRepository/Classes/Domain/Projection/Content/HierarchyTraversalDirection.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Neos\ContentRepository\Domain\Projection\Content;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * The hierarchy traversal direction value object
```

Comment:
Traversal in a (read-side) subgraph of NODES. -> find parent node -> UP; find child node -> DOWN